packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -71,25 +96,30 @@ export function SquadPageHeader({
       <SquadMemberShortList
         squad={squad}
         members={members}
-        className="my-6 laptopL:hidden"
+        className={classNames('my-6', !shouldUseListModeV1 && 'laptopL:hidden')}
       />
       <SquadHeaderBar
         squad={squad}
         members={members}
-        className="laptopL:absolute laptopL:right-18 laptopL:top-0"
+        className={classNames(
+          !shouldUseListModeV1 &&
+            'laptopL:absolute laptopL:right-18 laptopL:top-0',
+        )}
+        shouldUseListModeV1={shouldUseListModeV1}
       />
       <EnableNotification
         contentName={squad.name}
         source={NotificationPromptSource.SquadPage}
-        className={classNames('w-full', MAX_WIDTH)}
+        className={classNames('w-full', !shouldUseListModeV1 && MAX_WIDTH)}
       />
       <div
         className={classNames(
-          'relative bottom-0 flex w-full flex-col bg-background-default pt-8 tablet:absolute tablet:translate-y-1/2 tablet:flex-row tablet:p-0 laptopL:px-0',
+          'relative bottom-0 flex w-full flex-col bg-background-default pt-8 tablet:absolute tablet:translate-y-1/2 tablet:flex-row tablet:p-0',
+          !shouldUseListModeV1 && 'laptopL:px-0',
           shouldShowHighlightPulse && 'highlight-pulse',
-          allowedToPost
-            ? 'items-center justify-center laptop:max-w-[41.5rem]'
-            : 'laptop:max-w-[38.25rem]',
+          allowedToPost && 'items-center justify-center',
+          allowedToPost && !shouldUseListModeV1 && 'laptop:max-w-[41.5rem]',
+          !allowedToPost && !shouldUseListModeV1 && 'laptop:max-w-[38.25rem]',
```

Comment:
This feels like you could use ternary as well, but up to you?

Reply:
hm, it has 2 conditions the allowedToPost and shouldUseListModeV1 which would make the ternary expression a bit messy in my opinion